xivo_dao/helpers/db_views.py

```diff
@@ -65,15 +65,18 @@ def _create_table_from_selectable(name, selectable, indexes=None, **kwargs):
     indexes = indexes or []
     metadata = MetaData()
 
+    try:
+        columns = selectable.selected_columns  # SQLAlchemy >= 1.4
+    except AttributeError:
+        columns = selectable.columns  # SQLAlchemy 1.2 compat
+
```

Comment:
Next time, please don't spend time to make the code compatible and just pin sqlalchemy version.
This code is only intended to run on sqlalchemy v1.2 for now. so IMHO you have written dead code here ...
All good practice flagged in 1.4 that can be applied in 1.2 are good to do, but not more

Reply:
Got it